packages/transaction-controller/src/TransactionController.ts

```diff
@@ -3786,6 +3794,20 @@ export class TransactionController extends BaseController<
     this.#updateTransactionInternal(
       { transactionId, skipHistory: true },
       (txMeta) => {
+        const userFeeLevel = txMeta.userFeeLevel as GasFeeEstimateLevel;
+
+        if (Object.values(GasFeeEstimateLevel).includes(userFeeLevel)) {
```

Comment:
Minor, would it add readability if we put this in a boolean variable such as `isUsingGasFeeEstimateLevel`?